tests/py/test_markdown.py

```diff
@@ -6,50 +7,99 @@
 
 class TestMarkdown(Harness):
 
-    # render
+    # r - render
```

Comment:
In general I agree, but I think abbreviating is okay in this case. By now it's a fairly standard pattern in our test suite to abbreviate the name of the function we're testing in the block where we're testing it. In one sense this improves readability by allowing you to be more descriptive about the purpose of the test in the rest of the test name without going past our 100-char line limit. It also makes it easy to run the subset of the test suite dealing with those tests using `py.test -k test_r_`.